modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -0,0 +1,64 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-143068 - Verify a custom object entry with a date and time field can be created"
+	@priority = 5
+	test CanAddEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field and the time storage set to user input") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186246",
+				objectName = "CustomObject186246",
+				pluralLabelName = "Custom Objects 186246");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "DateTime",
+				fieldLabelName = "Custom Date Time",
+				fieldName = "customDateTime",
+				fieldType = "DateTime",
+				isRequired = "false",
+				objectName = "CustomObject186246",
+				timeStorageOption = "useInputAsEntered");
```

Comment:
Can we add the field using the UI here? We should have at least one test that does this